EIPS/blob-config.md

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
```suggestion
---
eip: 7840
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.